pkg/security/tests/syscall_tester.go

```diff
@@ -13,19 +13,12 @@ import (
 	"os"
 	"os/exec"
 	"testing"
-
-	"golang.org/x/sys/unix"
 )
 
 //go:embed syscall_tester/bin
 var syscallTesterFS embed.FS
 
 func loadSyscallTester(t *testing.T, test *testModule, binary string) (string, error) {
-	var uname unix.Utsname
```

Comment:
This looks fine to be deleted given where the syscall tester is most often used (CI and locally), where we know the architecture. There may be other use cases of the syscall tester that I'm not aware of, but we could revert this deletion when the time comes, and think about whether it's worth the added dependency.

Reply:
Looks like it was used to contain the results of the call to uname, and all it was checking was whether the call to uname succeeded

Reply:
Yeah, exactly. The actual values were never read, so there was no point in making this call.

Reply:
This would be useful for cases uname would fail. But idk where uname would fail (at least where syscall tester is run)

Reply:
uname shouldn't ever fail. It is a basic syscall to read system information like architecture, kernel release, etc.

Reply:
well, keyword "shouldn't"

Reply:
If it fails, so many other parts of system-probe are going to error out.

Reply:
thanks for the context!